### PR TITLE
Update wine_quality.csv filepath

### PR DESCRIPTION
**Pull request by HackyRoot**
## What changes are proposed in this pull request?

- Update the filepath of wine_quality.csv in [sklearn_elasticnet_wine](https://github.com/mlflow/mlflow/tree/master/examples/sklearn_elasticnet_wine), from local file to fetch it from UCI server. 
Resolves issue #1110  
 
## How is this patch tested?
 
N/A
 
### Is this a user-facing change? 

- [ ] No. You can skip the rest of this section.
- [X] Yes. Give a description of this change to be included in the release notes for MLflow users.
 
Instead of using local wine_quality.csv, fetch it from UCI server. Now users don't have to clone entire repo to try examples.
 
### What component(s) does this PR affect?
 
- [ ] UI
- [ ] CLI 
- [ ] API 
- [ ] REST-API 
- [X] Examples 
- [X] Docs
- [ ] Tracking
- [ ] Projects 
- [ ] Artifacts 
- [ ] Models 
- [ ] Scoring 
- [ ] Serving
- [ ] R
- [ ] Java
- [ ] Python

### How should the PR be classified in the release notes? Choose one:
 
- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [X] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: examples/sklearn_elasticnet_wine/train.py [hunk 1]
```diff
@@ -28,9 +28,11 @@ def eval_metrics(actual, pred):
     warnings.filterwarnings("ignore")
     np.random.seed(40)
 
-    # Read the wine-quality csv file (make sure you're running this from the root of MLflow!)
-    wine_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), "wine-quality.csv")
-    data = pd.read_csv(wine_path)
+    # Read the wine-quality csv file from the URL
+    try:
+        data = pd.read_csv('http://archive.ics.uci.edu/ml/machine-learning-databases/wine-quality/winequality-red.csv', sep=';')
+    except Exception as e:
```
Comment (smurching) [inline examples/sklearn_elasticnet_wine/train.py]: Could we instead use the Python `logging` module to still log the exception thrown by `read_csv`, e.g. as recommended [in this StackOverflow post](https://stackoverflow.com/a/31444861/12111347)?

**Review by smurching — COMMENTED** [review]
@HackyRoot generally looks good, just had one small comment

**Comment by HackyRoot** [comment]
@smurching thank you for the suggestion. I'll make the necessary changes.

File: examples/sklearn_elasticnet_wine/train.py (hunk 1 above)
Comment (HackyRoot) [reply examples/sklearn_elasticnet_wine/train.py]: Is this correct?
```
try:
     data = pd.read_csv('http://archive.ics.uci.edu/ml/machine-learning-databases/wine-quality/winequality-red.csv', sep=';')
    
except Exception as e:
        logger.exception("Unable to download training & test CSV, check your internet connection. Error: %s", e)
```

**Review by dbczumar — APPROVED** [review]
@HackyRoot LGTM. I made the changes suggested by @smurching and reverted some changes to the Jupyter notebook formatting. I'll merge this once the tests pass. Thank you for your contribution!

**Pull request merged**